exporters/geneva/include/opentelemetry/exporters/geneva/metrics/exporter.h

```diff
@@ -33,6 +33,21 @@ constexpr uint64_t kSecondsToUnixTime =
                   // 1601-01-01T00:00:00Z and UNIX/Linux epoch
                   // (1970-01-01T00:00:00Z)
 
+using ValueType = nostd::variant<int64_t, double>;
+
+enum class MetricsEventType : uint16_t {
```

Comment:
Is there a enum definition for default value like 0？

Reply:
`0` is defined as old_ifx value, but it is not something default 
https://github.com/open-telemetry/opentelemetry-cpp-contrib/blob/dba13fef65749f0fc703076a05896c7f28c44976/exporters/geneva/test/decoder/ifx_metrics_bin.ksy#L221-L223